src/Tribe/PUE/Checker.php

```diff
@@ -451,6 +448,27 @@ public function ajax_validate_key() {
 			exit;
 		}
 
+		/**
+		 * processes variable substitutions for server-side API message
+		 */
+		private function get_api_message( $info ) {
+			// this default message should never show, but is here as a fallback just in case.
+			$message = sprintf(
+				esc_html__( 'Sorry, there is a problem with your license key. You\'ll need to %scheck your license%s to have access to updates, downloads, and support.', 'tribe-common' ),
+				'+',//'<a href="https://theeventscalendar.com/license-keys/">',
```

Comment:
In case the fallback _is_ used, should we swap out `+` chars with the strings we need to form an actual link?